35/ex35.py

```diff
@@ -0,0 +1,95 @@
+#imports the exit command from systems as a way to end the game.
+from sys import exit
+
+# you finally enter the gold room after the bear room.
+def gold_room():
+    #The user is prompted to choose how much gold he wants.
+    print "This room is full of gold. How much do you take?"
+    choice = raw_input("> ")
+    # this try function converts the raw input into an integer, and if that does not
+    #output an error message as the user really typed in an integer, then it attributes that integer to the how_much variable.
+    try:
+        int(choice)
+        how_much = int(choice)
+    #if the user typed in a string then python converts the error message into a friendly message and ends the game.
+    except ValueError:
+        dead("Man, learn to type a number.")
+
+    if how_much < 50:
+        print "Nice, you're not greedy, you win!"
+        exit(0)
+    elif how_much > 10000:
+        dead("Unbelievable, we don't have enough gold")
+    else:
+        dead("You greedy bastard!")
+#Users enter the bear room if they picked the left door.
+def bear_room():
+    print "There is a bear here."
+    print "The bear has a bunch of honey."
+    print "The fat bear is in front of another door."
+    print "How are you going to move the bear?"
+    bear_moved = False
+    #infinite loop as long as it is true.
+    while True:
```

Comment:
Here's why `while True` works the way it does: http://stackoverflow.com/a/3754632